### PR TITLE
MK3_3.12: Add Menu Calibration -> Temp Model cal. 

### PR DESCRIPTION
**Pull request by 3d-gussner**
Runs M310 A F0
Enables with M310 S1
Saves settings M500

Tested on MK404 with Octoprint serial logging.

### CODE REVIEW

File: Firmware/temp_model.h [hunk 1]
```diff
@@ -59,6 +59,7 @@ struct model_data
 };
 
 static bool enabled;          // model check enabled
+static bool valid = false;    // model is valid
```
Comment (wavexx) [inline Firmware/temp_model.h:62]: Why is this needed? temp_model::calibrated() should do the same and avoid having to set/reset another state variable?

**Review by DRracer — APPROVED** [review]
